guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>Let's change the world imagery from input imagery parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>